newsroom/search.py

```diff
@@ -467,8 +467,8 @@ def prefill_search_products(self, search):
                 search.products = get_product_by_id(
                     search.args["product"],
                     product_type=search.section,
-                    company_id=search.company.get("_id"),
```

Comment:
you should still check if that product is assigned to user or a company, if not user shouldn't be able to see it